src/java.base/share/classes/java/time/Duration.java

```diff
@@ -719,8 +719,8 @@ public Duration plus(long amountToAdd, TemporalUnit unit) {
         if (amountToAdd == 0) {
             return this;
         }
-        if (unit instanceof ChronoUnit) {
-            switch ((ChronoUnit) unit) {
+        if (unit instanceof ChronoUnit u) {
+            switch (u) {
```

Comment:
Maybe `u` could be replaced with `chronoUnit` here too

Reply:
Variable name updated as suggested in 3dc1e07